applications/GeoMechanicsApplication/custom_constitutive/incremental_linear_elastic_interface_law.h

```diff
@@ -24,6 +25,17 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) GeoIncrementalLinearElasticInterface
 public:
     using BaseType = ConstitutiveLaw;
 
+    explicit GeoIncrementalLinearElasticInterfaceLaw(std::unique_ptr<ConstitutiveLawDimension> rConstitutiveLawDimension)
+        : mConstitutiveLawDimension(std::move(rConstitutiveLawDimension))
```

Comment:
Since the parameter is a smart pointer, I think its name should be (according to the Kratos Style Guide):
```suggestion
        : mConstitutiveLawDimension(std::move(pConstitutiveLawDimension))
```

Reply:
done